system-tests/lib/cre/don/config/definitions.go

```diff
@@ -188,6 +189,61 @@ func WorkerEVM(donBootstrapNodePeerID, donBootstrapNodeHost string, ocrPeeringDa
 	), nil
 }
 
+type WorkerSolanaInput struct {
+	Name                 string
+	ChainID              string
+	ChainSelector        uint64
+	NodeURL              string
+	FromAddress          solana.PublicKey
+	ForwarderAddress     string
+	ForwarderState       string
+	HasForwarderContract bool
+}
+
+func BootstrapSolana(chains []*WorkerSolanaInput) string {
+	var ret string
+	for _, chain := range chains {
+		ret += fmt.Sprintf(`
+		[[Solana]]
+		ChainID = '%s'
+		Enabled = true
+
+		[[Solana.Nodes]]
+		Name = '%s'
+		URL = '%s'
+		`, chain.ChainID, chain.Name, chain.NodeURL)
+	}
+
+	return ret
+}
+
+func WorkerSolana(chains []*WorkerSolanaInput) string {
+	var ret string
+	for _, chain := range chains {
+		ret += fmt.Sprintf(`
+		[[Solana]]
+		ChainID = '%s'
+		Enabled = true
+		TxRetentionTimeout = '5s'
+
+		[Solana.Workflow]
+		Enabled = true
+		ForwarderAddress = '%s'
+		FromAddress      = '%s'
+		ForwarderState   = '%s'
+		PollPeriod = '1s'
+		AcceptanceTimeout = '25s'
+		TxAcceptanceState = 3
+		Local = true
```

Comment:
please implement parametrisation of these values in the same way as it's done for writeEVM:
- default values in `configs/capability_defaults.toml` (things like `TxRetentionTimeout`, `PollPeriod`, `AcceptanceTimeout`, `TxAcceptanceState`, `Local`.
- allow to override these values per DON or even per-chain (if we think we should support more than 1 Solana chain)